var/spack/repos/builtin/packages/getorganelle/package.py

```diff
@@ -6,14 +6,15 @@
 from spack.package import *
 
 
-class PyGetorganelle(PythonPackage):
+class Getorganelle(PythonPackage):
```

Comment:
Instead of deleting the old package, you should first deprecate it: https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions